src/vet.test.ts

```diff
@@ -170,4 +173,131 @@ describe('Vet', () => {
       expect(result).toBe('')
     })
   })
+
+  describe('addStepSummary', () => {
+    // Mock core.summary
+    const mockSummary = {
+      clear: vi.fn().mockReturnThis(),
+      addRaw: vi.fn().mockReturnThis(),
+      write: vi.fn().mockResolvedValue(undefined)
+    }
+
+    beforeEach(() => {
+      // Reset all mocks
+      vi.clearAllMocks()
```

Comment:
```suggestion
      // Reset all mocks
      vi.resetAllMocks()
```

https://vitest.dev/api/vi.html#vi-resetallmocks